crowbar_framework/spec/fixtures/crowbar_maintenance.json

```diff
@@ -1,3 +1,4 @@
 {
-  "maintenance_updates_installed":true
+  "passed":true,
```

Comment:
agree that this is duplicated data, but more convenient for the API consumer. I'll check with @santiph (IIRC the requirement came from the UI)